go/signedexchange/README.md

```diff
@@ -137,9 +127,6 @@ As of July 2018, there are no CA that issues certificate with ["CanSignHttpExcha
 1. Host `example.org.hello.sxg` on a HTTPS server. Please see the previous section for details.
 
 1. Navigate to the signed exchange URL using a web browser supporting signed exchanges.
-    - As of July 2018, you can use Chrome M69 [Dev](https://www.google.com/chrome/?extra=devchannel)/[Canary](https://www.google.com/chrome/browser/canary.html) versions with the following two flags enabled:
-      - chrome://flags/#enable-signed-http-exchange
-      - chrome://flags/#allow-sxg-certs-without-extension
```

Comment:
I think it's still worthwhile to describe the chrome://flags/#allow-sxg-certs-without-extension flag until someone's offering free CanSignHttpExchanges certificates. The old text for it, of course, isn't right, and I don't mind getting the new text in a separate PR.

Reply:
Added #allow-sxg-certs-without-extension instructions back